.claude/skills/strings-xml-translation-check/scripts/translate-strings-xml.py

```
@@ -0,0 +1,297 @@
+                temperature=0,
+            ),
+        )
+        data = json.loads(resp.text)
+        if not isinstance(data, list) or len(data) != len(texts):
+            raise ValueError(
+                f"expected list of {len(texts)}, got "
+                f"{type(data).__name__} "
+                f"len={len(data) if isinstance(data, list) else 'n/a'}"
+            )
+        return [str(x) for x in data]
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)